unit/service/service.go

```diff
@@ -136,75 +146,6 @@ func (sv *Unit) Active() unit.Activation {
 	case Stop, StopSigabrt, StopPost, StopSigkill, StopSigterm, FinalSigkill, FinalSigterm:
 		return unit.Deactivating
 	default:
```

Comment:
what happens if you remove the `default: panic("unreachable")`?  Is the compiler smart enough to know those are the only possible cases?